Cargo.toml

```diff
@@ -28,6 +28,6 @@ debug = 0 # Set this to 1 or 2 to get more useful backtraces in debugger.
 # chalk-ir = { path = "../chalk/chalk-ir" }
 # chalk-recursive = { path = "../chalk/chalk-recursive" }
 
-# ungrammar = { path = "../ungrammar" }
+ungrammar = { path = "../ungrammar" }
```

Comment:
Including this in your changeset may be the cause of the current CI failures.

Reply:
Ye thats expected, will have to merge https://github.com/rust-analyzer/ungrammar/pull/36 first and then point to the updated version there